assets/src/amp-story-editor-blocks.js

```diff
@@ -52,9 +63,67 @@ domReady( () => {
 			maybeEnqueueFontStyle( block.attributes.ampFontFamily );
 		}
 	}
+
+	renderStoryComponents();
+
+	// Prevent WritingFlow component from focusing on last text field when clicking below the carousel.
+	document.querySelector( '.editor-writing-flow__click-redirect' ).remove();
 } );
 
+/**
+ * Add some additional elements needed to render our custom UI controls.
+ */
+function renderStoryComponents() {
+	const editorBlockList = document.querySelector( '.editor-block-list__layout' );
+	const postTitle = document.querySelector( '.editor-post-title' );
```

Comment:
It's going to happen at some point, but there's been a lack of traction and not a clear decision yet on how it's gonna happen. Since we probably want to require titles for stories anyway, I don't think there's an immediate need to change this. Definitely can keep this in mind though.

Reply:
Fixed now in 7e37eedd5aad17598f808c79f1b9c64265982b03, as I had to move the component out of the post title container in order to make the inserter work properly.